js/redirectorpage.js

```diff
@@ -116,24 +116,41 @@ function duplicateRedirect(index) {
 	saveChanges();
 }
 
+function checkIfGroupingExists() {
+	let grouping = REDIRECTS.map((row, i) => { return { row, index: i}})
+									.filter(result => result.row.grouped)
+									.sort((a, b) => a.index - b.index);
+	return grouping;
+}
+
 function toggleDisabled(index) {
-	let redirect = REDIRECTS[index];
-	redirect.disabled = !redirect.disabled
+	let grouping = checkIfGroupingExists();
+
+	if(grouping && grouping.length > 1) {
+		for (let redirect of grouping) {
+			let redirectDom = REDIRECTS[redirect.index];
+			redirectDom.disabled = !redirectDom.disabled
+			redirectDom.grouped = !redirectDom.grouped
+
+			let elm = document.querySelector("[data-index='" + (redirect.index).toString() + "']");
+			elm.classList.remove('grouped')
+			elm.childNodes[7].childNodes[3].classList.remove("checkMarked");
```

Comment:
`elm.childNodes[7].childNodes[3]` seems very fragile and hard to know what is being selected. Could you do this with a query selector instead, based on a tag name, or a classname?